drape/support_manager.cpp

```diff
@@ -15,14 +18,16 @@ void SupportManager::Init()
 {
   string const renderer = GLFunctions::glGetString(gl_const::GLRenderer);
   string const version = GLFunctions::glGetString(gl_const::GLVersion);
-  LOG(LINFO, ("Renderer =", renderer, "Version =", version));
-
-  // On Android the engine may be recreated. Here we guarantee that GPU info is sent once per session.
-  static bool gpuInfoSent = false;
-  if (!gpuInfoSent)
+  LOG(LINFO, ("Device =", GetPlatform().DeviceName(), "Renderer =", renderer, "Version =", version));
+  
+  // On Android support manager may be reinitialized. Here we guarantee that the info is sent once per session.
+  static bool supportManagerInfoSent = false;
+  if (!supportManagerInfoSent)
   {
     alohalytics::Stats::Instance().LogEvent("GPU", renderer);
-    gpuInfoSent = true;
+    alohalytics::Stats::Instance().LogEvent("CpuCores", {{GetPlatform().DeviceName(),
+                                                          strings::to_string(GetPlatform().CpuCores())}});
```

Comment:
I mean, hardware_concurrency() is only suggestion, not a real number of cpu cores available, and may vary during the application lifetime.